message-slack.mjs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+const env = process.env;
+
+const shortCommitHash = env.CIRCLE_SHA1.substring(0, 7);
+
+const response = await fetch('https://slack.com/api/chat.postMessage', {
+  method: 'POST',
+  headers: {
+    "Content-Type": "application/json",
+    "Authorization": "Bearer " + env.SLACK_ACCESS_TOKEN
+  },
+  body: JSON.stringify(
+    {
+      "channel": "deployments",
+      "blocks": [{
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": `*${env.CIRCLE_BRANCH}: Donate frontend deployed!* All visitors should see the updated app version within 30 minutes.\n` +
```

Comment:
I guess for `develop` we will lose the distinction between Staging & Regression here? Could it use the `env` job parameter that we already pass in for Jira?

Reply:
Sort of but we haven't been sending these messages for regression in the past, only for staging, and I'm continuing to only show it for staging. I don't think we really need the notifications for regression, it happens at the same time.